website/components/learn-nomad/index.jsx

```diff
@@ -0,0 +1,58 @@
+import Button from '@hashicorp/react-button'
+
+export default function LearnNomad({ items }) {
+  return (
+    <div className="g-learn-nomad">
+      <div className="g-grid-container">
+        <div className="column-container">
+          <div className="column info">
```

Comment:
I _think_ this wrapper div isn't needed... I don't see see `column` or `info` in styles.

Reply:
Ok, the div is used, but I think these class names are unused.

Reply:
Created a ticket for this here. We will handle post-launch!